datadog-monitors.tf

```
@@ -24,7 +24,7 @@ resource "datadog_monitor" "disk_space" {
   type    = "query alert"
   message = "@pagerduty"
 
-  query = "avg(last_5m):exclude_null(avg:system.disk.free{!device:tmpfs,!device:cgroup,!device:udev,!device:shm,!device:cgmfs,!label:UEFI,!label:uefi,!device:/dev/loop*} by {host,device}) / exclude_null(avg:system.disk.total{!device:tmpfs,!device:cgroup,!device:udev,!device:shm,!device:cgmfs,!label:UEFI,!label:uefi,!device:/dev/loop*} by {host,device}) * 100 < 10"
+  query = "avg(last_10m):exclude_null(avg:system.disk.free{!device:tmpfs,!device:cgroup,!device:udev,!device:shm,!device:cgmfs,!label:UEFI,!label:uefi,!device:/dev/loop*} by {host,device}) / exclude_null(avg:system.disk.total{!device:tmpfs,!device:cgroup,!device:udev,!device:shm,!device:cgmfs,!label:UEFI,!label:uefi,!device:/dev/loop*} by {host,device}) * 100 < 10"
```

```suggestion
  query = "avg(last_15m):exclude_null(avg:system.disk.free{!device:tmpfs,!device:cgroup,!device:udev,!device:shm,!device:cgmfs,!label:UEFI,!label:uefi,!device:/dev/loop*} by {host,device}) / exclude_null(avg:system.disk.total{!device:tmpfs,!device:cgroup,!device:udev,!device:shm,!device:cgmfs,!label:UEFI,!label:uefi,!device:/dev/loop*} by {host,device}) * 100 < 10"
```
10 min should be enough for most builds, but increasing it to 15 min shouldn't make a lot of difference while letting plenty of time to alert before filling disks.